baseplate/core.py

```diff
@@ -187,6 +190,7 @@ def configure_tracing(self, service_name,
                 num_span_workers=num_span_workers,
                 span_batch_interval=span_batch_interval,
                 sample_rate=sample_rate,
+                log_if_unconfigured=log_if_unconfigured,
```

Comment:
I was thinking about that, but then I convinced myself that any application using this stuff would be manually calling `configure_tracing`, at which point it has to handle the config itself. Is that not right?